<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editticketdialog</class>
 <widget class="QDialog" name="editticketdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Uni Sans Heavy CAPS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(2, 173, 237);</string>
   </property>
   <property name="text">
    <string>EDIT TICKET SUBMISSION</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>401</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string> TICKET DETAILS</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>105</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lb_ticket_num">
       <property name="text">
        <string>Ticket Number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_incident_cat">
       <property name="text">
        <string>Incident Category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_tags">
       <property name="text">
        <string>Tags</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_impact">
       <property name="text">
        <string>Impact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_urgency">
       <property name="text">
        <string>Urgency</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_priority">
       <property name="text">
        <string>Prioity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_timestamp">
       <property name="text">
        <string>Timestamp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_level_support">
       <property name="text">
        <string>Level of Support</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_5"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_4"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_6"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_3"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>CONTACT DETAILS</string>
   </property>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lb_name">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_email">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_phone">
       <property name="text">
        <string>Phone</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_11"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_12"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_symptoms_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>103</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SYMPTOMS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRating">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Rating</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
